e2e/cephfs.go

```diff
@@ -258,6 +258,34 @@ var _ = Describe("cephfs", func() {
 					}
 				})
 
+				By("validate multiple subvolumegroup creation", func() {
+					// re-define configmap with information of multiple clusters.
+					deleteResource(cephfsExamplePath + "storageclass.yaml")
+					createCustomConfigMap(cephfsDirPath, f.ClientSet, "clusterID-1", "clusterID-2")
+					createCephfsStorageClass(f.ClientSet, f, false, "clusterID-1")
+					validatePVCAndAppBinding(pvcPath, appPath, f)
+					deleteResource(cephfsExamplePath + "storageclass.yaml")
+
+					// create resources for the second cluster.
+					createCephfsStorageClass(f.ClientSet, f, false, "clusterID-2")
+					validatePVCAndAppBinding(pvcPath, appPath, f)
+					deleteResource(cephfsExamplePath + "storageclass.yaml")
+
+					// verify multiple subvolumegroup creation.
```

Comment:
move 1st validation to line 267 to keep tests in order

Reply:
Thanks, I'll make the changes.